cl/assets/tailwind/tailwind.config.js

```diff
@@ -22,14 +22,19 @@ module.exports = {
   theme: {
     extend: {
       screens: {
-        xs: '380px',
+        xs: '392px',
       },
       spacing: {
         4.5: '18px',
         7.5: '30px',
         13: '52px',
         15: '60px',
+        18: '72px',
         35: '140px',
+        41: '164px',
+        42: '168px',
+        53: '212px',
+        70: '280px',
```

Comment:
We should probably use `rem` for these spacing values. I noticed that most Tailwind classes use `rem` instead of `px`. At first, I thought the new spacing might be following a different convention, but it looks like you're using the dafault tailwind spacing scale (one unit is equal to `0.25rem`, which translates to `4px`). So, I think it would be consistent to use rem here as well.